tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs

```diff
@@ -19,8 +19,11 @@ namespace FamilyHubs.ReferralUi.UnitTests.Pages.ProfessionalReferral;
 
 public class WhenUsingLocalOfferDetail
 {
-    [Fact]
-    public async Task ThenOnGetAsync_LocalOfferDetailWithReferralNotEnabled()
+    [Theory]
+    [InlineData(default!)]
+    [InlineData("url")]
+    [InlineData("https://wwww.google.com")]
```

Comment:
Please could you add test for http://google.com

Reply:
Added as requested